app/mailers/publisher_notifications.rb

```diff
@@ -0,0 +1,16 @@
+class PublisherNotifications < Mail::Notify::Mailer
```

Comment:
I think the convention is that `PublisherNotifications` (and all mailers) should inherit from `ApplicationMailer`, and then `ApplicationMailer` can be changed to inherit from `Mail::Notify::Mailer` — but again, not sure if this is important enough to warrant changing.